tensorflow/core/kernels/conv_grad_filter_ops.cc

```diff
@@ -55,7 +55,7 @@ limitations under the License.
 #endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 #if GOOGLE_CUDA
 #include "tensorflow/stream_executor/cuda/ptxas_utils.h"
-#include "tensorflow/stream_executor/cuda/redzone_allocator.h"
+#include "tensorflow/stream_executor/redzone_allocator.h"
```

Comment:
That's right! In places where `#if` are still allowed, I'd prefer it to be included in the include-guard though, just so that one from `ROCm` side doesn't mistakenly used `redzone_allocator` thinking it is ready and only to figure out it is not when running.